### ISSUE TITLE
Regression: Taiko not finding all text elements in a given page.

### ISSUE THREAD

**Issue by nehashri** [getgauge/taiko#571]
Taiko doesn't try to find all the elements in a given page anymore. 

**Steps to Reproduce**
Run taiko with the given script
```
const { openBrowser, goto, text, closeBrowser } = require('taiko');
(async () => {
    try {
        await openBrowser();
        await goto("github.com");
        await text('Sign up').get().then(elements => console.log(elements.length));
    } catch (e) {
        console.error(e);
    } finally {
        await closeBrowser();
    }
})();
```
Running it with the latest taiko version `4cc1561abd50491e5a6165acfb31d0441e32ca16` gives the following output.
``` ✔ Browser opened
 ✔ Navigated to url http://github.com
1
 ✔ Browser closed
```

However, running the script with taiko version `f86cf3ffd43` i.e. `0.6.0` gives the output
```
✔ Browser opened
 ✔ Navigated to url "http://github.com"
5
 ✔ Browser closed
```

Notice that only 1 element was found with latest version as compared to 5 in 0.6.0 version.

**Comment by shubhamsc** [comment]
Fix availables in 574e989dc09432ddcf4494e44257d63da4c8bf53

In [Github](http://github.com) page, The text `Sign up` has exactly one match but if we try to find contains search with text `sign`, it will match all texts node in the page which appears 8 times.

**Comment by Debashis9012** [comment]
This issue has been verified and found fixed with the commit version ```574e989 ```
**Note:**
a. If the text value matched with the exact string it will return the total count of exact matching string.
b. If the text did not match with the exact string then it will return the total count of partial matching string.
c. If the text is available inside the text/input box then the text value should be counted or considered for exact/partial match.
